File: server-scala-play-server/app/model/WordzWordResponse.scala
package model

import play.api.libs.json._

/**
  * Represents the Swagger definition for WordzWordResponse.
  */
@javax.annotation.Generated(value = Array("org.openapitools.codegen.languages.ScalaPlayFrameworkServerCodegen"), date = "2026-02-09T20:57:39.376804970Z[Etc/UTC]", comments = "Generator version: 7.20.0-SNAPSHOT")
case class WordzWordResponse(
  id: Option[Long],
  word: Option[String],
  definition: Option[String],
  scores: Option[ScoreListResponse],
  icon: Option[AssetShortResponse],
  image: Option[AssetShortResponse],
  authorOverride: Option[String],
  updatedDate: Option[Long],
  startDate: Option[Long],
  endDate: Option[Long],
  createdDate: Option[Long],
  active: Option[Boolean],
  allocateTickets: Option[Boolean],
  ticketType: Option[String],
  ticketCount: Option[Long],
  points: Option[Long]
)

object WordzWordResponse {
  implicit lazy val wordzWordResponseJsonFormat: Format[WordzWordResponse] = Json.format[WordzWordResponse]
}

File: server-aspnet-fastendpoints/src/Org.OpenAPITools/Models/WordzWordResponse.cs
namespace Org.OpenAPITools.Models;


/// <summary>
/// 
/// </summary>
public class WordzWordResponse 
{
    public long Id { get; set; }
    public string Word { get; set; }
    public string Definition { get; set; }
    public ScoreListResponse Scores { get; set; }
    public AssetShortResponse Icon { get; set; }
    public AssetShortResponse Image { get; set; }
    public string AuthorOverride { get; set; }
    public long UpdatedDate { get; set; }
    public long StartDate { get; set; }
    public long EndDate { get; set; }
    public long CreatedDate { get; set; }
    public bool Active { get; set; }
    public bool AllocateTickets { get; set; }
    public string TicketType { get; set; }
    public long TicketCount { get; set; }
    public long Points { get; set; }
}


